instrumentation/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/ApplicationOpenTelemetry.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opentelemetryapi;
+
+import application.io.opentelemetry.api.OpenTelemetry;
+import application.io.opentelemetry.api.OpenTelemetryBuilder;
+import application.io.opentelemetry.api.metrics.MeterProvider;
+import application.io.opentelemetry.api.trace.TracerProvider;
+import application.io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.javaagent.instrumentation.opentelemetryapi.context.propagation.ApplicationContextPropagators;
+import io.opentelemetry.javaagent.instrumentation.opentelemetryapi.metrics.ApplicationMeterProvider;
+import io.opentelemetry.javaagent.instrumentation.opentelemetryapi.trace.ApplicationTracerProvider;
+
+public class ApplicationOpenTelemetry implements OpenTelemetry {
+
+  public static final OpenTelemetry INSTANCE = new ApplicationOpenTelemetry();
+
+  private final TracerProvider applicationTracerProvider;
+  private final MeterProvider applicationMeterProvider;
+
+  private final ContextPropagators applicationContextPropagators;
+
+  private ApplicationOpenTelemetry() {
+    io.opentelemetry.api.OpenTelemetry agentOpenTelemetry =
+        io.opentelemetry.api.OpenTelemetry.get();
+    applicationTracerProvider =
+        new ApplicationTracerProvider(agentOpenTelemetry.getTracerProvider());
+    applicationMeterProvider = new ApplicationMeterProvider(agentOpenTelemetry.getMeterProvider());
+    applicationContextPropagators =
+        new ApplicationContextPropagators(agentOpenTelemetry.getPropagators());
+  }
+
+  @Override
+  public void setPropagators(ContextPropagators contextPropagators) {
+    throw new UnsupportedOperationException("To be implemented?");
```

Comment:
I don't think we supported this ever, it would just no-op into the application OpenTelemetry which is never used. I guess we would need an AgentContextPropatators(application.ContextPropagators) bridge class if we wanted to?